### DISCUSSION TITLE
SPX v1.15.1-rc5 - Testing & Call Outs

### DISCUSSION THREAD

**Discussion by Reiers** [filecoin-project/lotus#8419] [Show and tell]
![SPX_TEST](https://user-images.githubusercontent.com/63351350/161642788-cd133291-782c-4941-b1d0-560901eb19d8.png)

### Hello SPX Fellows

Update your SPs to Lotus [v1.15.1-rc5](https://github.com/filecoin-project/lotus/releases/tag/v1.15.1).

🚀 This post if you are participating in this testing round. 
If you aren't able to join for reasons, please reach out to @jennijuju or me on Slack!

## Regular Operations 

Your daily SP operations should stay ✅ .

**Reply with:**
- 🦭If you have successfully sealed some new sectors.
- 🤝If you have successfully made some storage deals.
- 🔥 If you have successfully made some snap deals.
- 🗄 If you have successfully setup and config the Index Provider.
- 📂 If you have successfully had some retrieval deals.
- 💻 If you passed your first WindowPoSt.

❗ Create [an issue](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Fbug&template=bug_report.yml) if you think there is a bug, don't forget to attach the logs!
   - Save the daemon logs, miner logs, and `lotus-miner info all` if you want to report a bug! If the log files are large, you can upload them in `SPX - v1.15.1-rc5 Logs` folder and link it in your issue report/slack message/..

## New feature and Improvements

- Read about the new Snap Deal queue, [here](https://lotus.filecoin.io/storage-providers/operate/snap-deals/#snap-deal-queue). 
- A LOT of Snap Deal bug fixes.
- Snap Deals support in `lotus-miner storage find <sectornumb>`, now you will find the location of the original CC and where the new data is located. 
- `lotus-miner sectors list --initial-pledge` is now a thing, give it try - it does what it says.
- Run `lotus-miner config default`, compare it to your `config.toml` - lots of things have been added. Its a good practice to do every now and then.

See more Improvements & Bug fixes see [here.](https://github.com/filecoin-project/lotus/releases/tag/v1.15.1-rc5)

**Comment by Reiers** [comment]
Reply Template: 

- [ ] 🦭If you have successfully sealed some new sectors.
- [ ] 🤝If you have successfully made some storage deals.
- [ ] 🔥 If you have successfully made some snap deals.
- [ ] 🗄 If you have successfully setup and config the Index Provider.
- [ ] 📂 If you have successfully had some retrieval deals.
- [ ] 💻 If you passed your first WindowPoSt.

**Reply by Reiers** [reply]
- [x] 🦭If you have successfully sealed some new sectors.
- [x] 🤝If you have successfully made some storage deals.
- [x] 🔥 If you have successfully made some snap deals.
- [x] 🗄 If you have successfully setup and config the Index Provider.
- [x] 📂 If you have successfully had some retrieval deals.
- [x] 💻 If you passed your first WindowPoSt.

**Comment by SBudo** [comment]
Upgraded to v1.15.1-rc5 24 hours ago (05 April 2022 @ 21:09 AEST)

Results:

- [x] If you have successfully sealed some new sectors.
- [x] 🤝If you have successfully made some storage deals.
- [x] 🔥 If you have successfully made some snap deals.
- [x] 🗄 If you have successfully setup and config the Index Provider.
- [ ]  📂 If you have successfully had some retrieval deals.
- [x] 💻 If you passed your first WindowPoSt.

**Comment by TippyFlitsUK** [comment]
- [x] 🦭If you have successfully sealed some new sectors.
- [x] 🤝If you have successfully made some storage deals.
- [x] 🔥 If you have successfully made some snap deals.
- [x] 🗄 If you have successfully setup and config the Index Provider.
- [x] 📂 If you have successfully had some retrieval deals.
- [x] 💻 If you passed your first WindowPoSt.

**Comment by neondragon** [comment]
- [ ] 🦭If you have successfully sealed some new sectors.
- [ ] 🤝If you have successfully made some storage deals.
- [ ] 🔥 If you have successfully made some snap deals.
- [x] 🗄 If you have successfully setup and config the Index Provider.
- [ ] 📂 If you have successfully had some retrieval deals.
- [x] 💻 If you passed your first WindowPoSt.

**Comment by tmyuu** [comment]
* [x]  🦭If you have successfully sealed some new sectors.
* [x]  🤝If you have successfully made some storage deals.
* [x]  🔥 If you have successfully made some snap deals.
* [x]  🗄 If you have successfully setup and config the Index Provider.
* [x]  📂 If you have successfully had some retrieval deals.
* [x]  💻 If you passed your first WindowPoSt.

**Comment by Angelo-gh3990** [comment]
> Reply Template:
> 
> * [x]  🦭If you have successfully sealed some new sectors.
> * [x]  🤝If you have successfully made some storage deals.
> * [x]  🔥 If you have successfully made some snap deals.
> * [x]  🗄 If you have successfully setup and config the Index Provider.
> * [x]  📂 If you have successfully had some retrieval deals.
> * [x]  💻 If you passed your first WindowPoSt.

**Comment by benjaminh83** [comment]
- [x] 🦭 If you have successfully sealed some new sectors.
- [x] 🤝 If you have successfully made some storage deals.
- [ ] 🔥 If you have successfully made some snap deals.
- [x] 🗄 If you have successfully setup and config the Index Provider.
- [x] 📂 If you have successfully had some retrieval deals.
- [x] 💻 If you passed your first WindowPoSt.

**Comment by jamerduhgamer** [comment]
- [x]  🦭If you have successfully sealed some new sectors.
- [ ]  🤝If you have successfully made some storage deals.
- [ ]  🔥 If you have successfully made some snap deals.
- [X]  🗄 If you have successfully setup and config the Index Provider.
- [ ]  📂 If you have successfully had some retrieval deals.
- [X]  💻 If you passed your first WindowPoSt.

**Comment by cryptowhizzard** [comment]
> * [X]  🦭If you have successfully sealed some new sectors.
> * [X]  🤝If you have successfully made some storage deals.
> * [X]  🔥 If you have successfully made some snap deals.
> * [X]  🗄 If you have successfully setup and config the Index Provider.
> * [X]  📂 If you have successfully had some retrieval deals.
> * [X]  💻 If you passed your first WindowPoSt.

**Comment by Meest-NAM** [comment]
# Meest NAM
- [X] 🦭If you have successfully sealed some new sectors.
- [X] 🤝If you have successfully made some storage deals.
- [X] 🔥 If you have successfully made some snap deals.
- [X] 🗄 If you have successfully setup and config the Index Provider.
- [X] 📂 If you have successfully had some retrieval deals.
- [X] 💻 If you passed your first WindowPoSt.
